app/frontend/src/application/jobseekers/manageQualifications.test.js

```diff
@@ -0,0 +1,128 @@
+import manageQualifications, {
+  addAddSubjectEventListener,
+  addSubjectHandler,
+  renumberRow,
+  renumberRows,
+  deleteRowHandler,
+  DELETE_BUTTON_CLASSNAME,
+  FIELDSET_CLASSNAME,
+  GOVUK_INPUT_CLASSNAME,
+  ROW_CLASS,
+  SUBJECT_LINK_ID,
+} from './manageQualifications';
+
+describe('manageQualifications', () => {
+  const originalMarkup = `<div class="${FIELDSET_CLASSNAME}">
```

Comment:
I don't think we need any of this anymore, no?

Reply:
this is markup for tests? yes need that if thats what you mean?

Reply:
Ah it's a test 🤦  Never mind me!